gatsby-browser.js

```diff
@@ -4,4 +4,12 @@
  * See: https://www.gatsbyjs.com/docs/browser-apis/
  */
 
-import './src/styles/global.css';
\ No newline at end of file
+import React from "react"
+
+import { EnrollmentProvider } from "./src/context/enrollment"
+
+import "./src/styles/global.css"
+
+export const wrapRootElement = ({ element }) => (
+  <EnrollmentProvider>{element}</EnrollmentProvider>
```

Comment:
Should we use this in the root?

Reply:
Yes, that's the way React context works and this is the recommended method.